packages/xo-web/src/xo-app/vm/tab-network.js

```diff
@@ -302,6 +309,244 @@ class VifStatus extends BaseComponent {
+                <Col size={6}>
+                  <Toggle onChange={this.toggleState('allow')} value={allow} />
+                </Col>
+              </SingleLineRow>
+              <SingleLineRow>
+                <Col size={6}>{_('aclRuleProtocol')}</Col>
+                <Col size={6}>
+                  <Select
+                    name='protocol'
+                    onChange={this.linkState('protocol')}
+                    options={[
+                      { label: 'IP', value: 'IP' },
+                      { label: 'TCP', value: 'TCP' },
+                      { label: 'UDP', value: 'UDP' },
+                      { label: 'ICMP', value: 'ICMP' },
+                      { label: 'ARP', value: 'ARP' },
+                    ]}
+                    value={protocol}
+                  />
+                </Col>
+              </SingleLineRow>
+              {showPort && (
+                <SingleLineRow>
+                  <Col size={6}>{_('aclRulePort')}</Col>
+                  <Col size={6}>
+                    <input
+                      className='form-control'
+                      min='1'
+                      onChange={this.linkState('port')}
+                      type='number'
+                      value={port}
+                    />
+                  </Col>
+                </SingleLineRow>
+              )}
+              {showIpRange && (
+                <SingleLineRow>
+                  <Col size={6}>{_('aclRuleIpRange')}</Col>
+                  <Col size={6}>
+                    <input
+                      className='form-control'
+                      key='key'
+                      onChange={this.linkState('ipRange')}
+                      type='text'
+                      value={ipRange}
+                    />
+                  </Col>
+                </SingleLineRow>
+              )}
+              <SingleLineRow>
+                <Col size={6}>{_('aclRuleDirection')}</Col>
+                <Col size={6}>
+                  <Select
+                    name='direction'
+                    onChange={this.linkState('direction')}
+                    options={[
+                      { label: 'from', value: 'from' },
+                      { label: 'to', value: 'to' },
+                      { label: 'from/to', value: 'from/to' },
+                    ]}
+                    required
+                    value={direction}
+                  />
+                </Col>
+              </SingleLineRow>
+            </Container>
+          </div>
+        </fieldset>
+      </form>
+    )
+  }
+}
+
+@addSubscriptions({
+  plugins: subscribePlugins,
+})
+class AclRuleRow extends Component {
+  render() {
+    const { rule, vif, plugins } = this.props
+    const ruleObj = JSON.parse(rule)
+    const sdnControllerLoaded = plugins.some(
+      plugin => plugin.name === 'sdn-controller' && plugin.loaded
+    )
+
+    return (
+      <tr>
+        <td>{ruleObj.allow ? 'Enable' : 'Disable'}</td>
```

Comment:
Enable*d* and disable*d*?

Reply:
Yes so I'll use `stateEnabled` and `stateDisabled` that already exist